Carbon.Registry/Functions/Grant-CRegistryPermission.ps1

```diff
@@ -0,0 +1,263 @@
+
+function Grant-CRegistryPermission
+{
+    <#
+    .SYNOPSIS
+    Grants permission on a file, directory, registry key, or certificate's private key/key container.

```

Comment:
Is this supposed to mention file, directory, and certificate paths (i.e. non registry paths)?